tox.ini

```diff
@@ -11,5 +11,4 @@ envlist =
 
 [testenv]
 commands = pytest {posargs}
-deps =
-    six,pytest,freezegun
+deps = -rtest_requirements.txt
```

Comment:
I know I'm late here but you'd usually see requirements-dev.txt rather than test_requirements.

It is even "reserved" in PyPI https://pypi.python.org/pypi/requirements-dev.txt

Reply:
Definitely not too late to move it. What do you think of the suggestion of dropping `six` proposed above? We can do both changes in one PR.

Reply:
Your requirement file should provide a "reproducible python environment". It should include all your dependencies.
What some people do if you want to split runtime/build vs test requirements you can include one from another.

Example: having `requirements.txt` with actual dependencies and `requirements-dev.txt` including requirements.txt (you can do -r)

Reply:
What you might want though is to pin the versions in your requirements file.

Reply:
You might be able to add mock using [markers](https://www.python.org/dev/peps/pep-0426/#environment-markers) as well here:
`mock ; python_version < '3.0'` (Try this out, I am just writing in github)